extras/rya.console/src/main/java/mvm/rya/shell/RyaConnectionCommands.java

```diff
@@ -164,8 +162,8 @@ public String connectToMongo(
             final MongoClient client = new MongoClient(new ServerAddress(hostname), Lists.newArrayList(creds));
 
             // Initialize the connected to Mongo shared state.
-            final MongoConnectionDetails accumuloDetails = new MongoConnectionDetails(username, password, instanceName, hostname);
-            final RyaCommands commands = RyaCommands.buildMongoCommands(accumuloDetails, client);
+            final MongoConnectionDetails mongoDetails = new MongoConnectionDetails(username, password, instanceName, hostname);
```

Comment:
If this is also going to connect to an instance of Rya, then you need to update the shared shell state to indicate it's connected to an instance. However, this deviates from how the Accumulo side works, so this may cause user's confusion.